mintapi/signIn.py

```diff
@@ -46,27 +46,23 @@ def get_email_code(
     try:
         imap_client = imaplib.IMAP4_SSL(imap_server)
     except imaplib.IMAP4.error:
-        logger.error("ERROR: Unable to establish IMAP Client")
-        return ""
+        raise RuntimeError("Unable to establish IMAP Client")
 
     try:
         rv, data = imap_client.login(imap_account, imap_password)
     except imaplib.IMAP4.error:
-        logger.error("ERROR: email login failed")
-        return ""
+        raise RuntimeError("Unable to login to IMAP Email")
 
     code = ""
     for c in range(20):
         time.sleep(10)
         rv, data = imap_client.select(imap_folder)
         if rv != "OK":
-            logger.error("ERROR: Unable to open mailbox ", rv)
-            return ""
+            raise RuntimeError("Unable to open mailbox: " + rv)
 
         rv, data = imap_client.search(None, "ALL")
         if rv != "OK":
-            logger.error("ERROR: Email search failed")
-            return ""
+            raise RuntimeError("Unable to search the Email folder")
```

Comment:
Sometimes we add `rv` to the error message and sometimes not. Would it make sense to always include it like on line 61? I realize you are just converting them as-is; I'm just curious if it can be improved even a little more :)

Reply:
Yup, I can do that.